frontend/src/components/worksheets/items/ImageItem.js

```diff
@@ -27,12 +53,28 @@ class ImageItem extends React.Component {
         if (this.props.item.hasOwnProperty('width')) {
             styles['width'] = this.props.item.width + 'px';
         }
-
         return (
-            <div className='ws-item' onClick={this.handleClick}>
-                <div className={className} ref={this.props.item.ref}>
+            <div className='ws-item'>
+                <div className={className} ref={this.props.item.ref} onClick={this.handleClick}>
```

Comment:
Why did you move the `onClick` handler to the child?

Reply:
Otherwise the click will trigger when we are clicking on the bundle detail